config/express.js

```diff
@@ -47,7 +47,7 @@ if (config.env === 'development') {
 }
 
 // mount all routes on /api path
-app.use('/api', routes);
+app.use('/', routes);
```

Comment:
👍  I have done the exact same thing in my PR.I was not sure how strongly you guys felt about having the word api in all endpoints.